readthedocs/api/base.py

```diff
@@ -223,7 +221,11 @@ def get_anchor(self, request, **kwargs):
         self.throttle_check(request)
 
         query = request.GET.get('q', '')
-        redis_data = redis_client.keys("*redirects:v4*%s*" % query)
+        try:
+            redis_client = cache.get_client(None)
```

Comment:
Seems like it should just be using `django.core.cache.cache`. Seems this code would be dependent on implementation of the redis backend.